src/main/java/br/unb/cic/analysis/ioa/InterproceduralOverrideAssignment.java

```diff
@@ -1,56 +1,299 @@
 package br.unb.cic.analysis.ioa;
 
+import br.unb.cic.analysis.AbstractAnalysis;
 import br.unb.cic.analysis.AbstractMergeConflictDefinition;
+import br.unb.cic.analysis.df.DataFlowAbstraction;
+import br.unb.cic.analysis.model.Conflict;
+import br.unb.cic.analysis.model.Statement;
 import soot.*;
+import soot.jimple.AssignStmt;
+import soot.jimple.InvokeStmt;
+import soot.jimple.internal.JInstanceFieldRef;
+import soot.toolkits.scalar.ArraySparseSet;
+import soot.toolkits.scalar.FlowSet;
 
 import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
 
-public class InterproceduralOverrideAssignment extends SceneTransformer  {
-
-    private Set<SootMethod> visitedMethods;
+// TODO Add treatment of if, loops ... (ForwardFlowAnalysis)
+// TODO Do not add anything when assignments are equal.
+public class InterproceduralOverrideAssignment extends SceneTransformer implements AbstractAnalysis {
 
+    private Set<Conflict> conflicts;
     private PointsToAnalysis pta;
-
     private AbstractMergeConflictDefinition definition;
+    private FlowSet<DataFlowAbstraction> res;
+    private Body body;
+
+    private Logger logger;
 
     public InterproceduralOverrideAssignment(AbstractMergeConflictDefinition definition) {
-        visitedMethods = new HashSet<>();
+        this.conflicts = new HashSet<>();
         this.definition = definition;
+        this.res = new ArraySparseSet<>();
+
+        this.logger = Logger.getLogger(
+                InterproceduralOverrideAssignment.class.getName());
+    }
+
+    @Override
+    public void clear() {
+        conflicts.clear();
+    }
+
+    @Override
+    public Set<Conflict> getConflicts() {
+        return conflicts;
+    }
+
+    private void configureEntryPoints() {
+        List<SootMethod> entryPoints = new ArrayList<>();
+        definition.getSourceStatements().forEach(s -> entryPoints.add(s.getSootMethod()));
+        Scene.v().setEntryPoints(entryPoints);
     }
 
     @Override
     protected void internalTransform(String s, Map<String, String> map) {
         definition.loadSourceStatements();
         definition.loadSinkStatements();
+        List<SootMethod> traversedMethods = new ArrayList<>();
```

Comment:
@barbosamaatheus qual a motivação para definir traversedMethods como uma variável local ao invés de um atributo da classe (como o visitedMethods de antes)?

Reply:
Como eu sempre passo os métodos que j foram visitados como parâmetro no método travese eu decidi por deixar assim. Mas na verdade antes (na casca de Rodrigo) existia o atributo e a variável local. Eu apenas escolhi um deles. O melhor seria usar como atributo mesmo?

Reply:
como atributo seria util se depois de executar a analise, por exemplo, voce precise listar que métodos foram analisados. se isso ainda nao for necessário, deixa como esta mesmo.